src/libsyntax/codemap.rs

```diff
@@ -592,8 +592,30 @@ impl CodeMap {
         }
     }
 
-    /// Given a `Span`, try to get a shorter span ending just after the first
-    /// occurrence of `char` `c`.
+    /// Given a `Span`, get a new `Span` covering the first token and all its trailing whitespace or
+    /// the original `Span`.
+    ///
+    /// If `sp` points to `"let mut x"`, then a span pointing at `"let "` will be returned.
+    pub fn span_until_non_whitespace(&self, sp: Span) -> Span {
+        if let Ok(snippet) = self.span_to_snippet(sp) {
+            let mut offset = 1;
```

Comment:
This should start at 0. I believe this is what caused CI to fail because if `snippet` is `"MallocSizeOf"` it will panic the same way: [playground](https://play.rust-lang.org/?gist=6ba340f27a0954682b9ac6bbe77f64c9&version=stable).